From 76015263b882a667e4f17bf54e8e0a49fbcf5af2 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Mon, 22 Apr 2013 18:29:53 +0300
Subject: [PORT FROM MAIN] Telephony: SPN name not initialized.

BZ: 11860

Old spn record cleared upon instantiation and disposal
of SIMRecords.

Change-Id: If8b6a677d9a4012459760aa7759f3fe721f7593f
Orig-Change-Id: I614479d05d92f8017e7039d6feecdf4a79f8bfcb
Category: aosp improvement
Domain: Telephony-Telephony-FWK
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 .../internal/telephony/uicc/SIMRecords.java        |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/java/com/android/internal/telephony/uicc/SIMRecords.java b/src/java/com/android/internal/telephony/uicc/SIMRecords.java
index 721758a..9be2da1 100644
--- a/src/java/com/android/internal/telephony/uicc/SIMRecords.java
+++ b/src/java/com/android/internal/telephony/uicc/SIMRecords.java
@@ -230,6 +230,7 @@ public class SIMRecords extends IccRecords {
         mIccId = null;
         // -1 means no EF_SPN found; treat accordingly.
         mSpnDisplayCondition = -1;
+        mSpn = null;
         mEfMWIS = null;
         mEfCPHS_MWI = null;
         mSpdiNetworks = null;
-- 
1.7.4.1

